### ISSUE TITLE
Chart files should match chart name

### ISSUE THREAD

**Issue by strangiato** [openshift-helm-charts/charts#1614]
It looks like when indexing the charts, the charts are currently appending `redhat-` to the front of the chart name, which unfortunately breaks helm dependencies feature.

For example, if I create the following `Chart.yaml` file:

```
apiVersion: v2
name: my-chart
description: A Helm example
type: application

version: 0.1.0

dependencies:
  - name: redhat-postgresql-persistent
    version: "0.0.3"
    repository: "https://charts.openshift.io"
```

And run the following to pull the dependent chart:

```
helm dependency build .
```

The chart successfully resolves but it downloads the file `postgresql-persistent-0.0.3.tgz`.  Then when attempting to deploy the chart it complains that the dependencies aren't up to date because it is looking for a file called `redhat-postgresql-persistent-0.0.3.tgz`.  When running a `helm install` it generates the following:

```
Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: redhat-postgresql-persistent
```

**Comment by komish** [comment]
@strangiato my apologies for the delay in response. What you're seeing is a side effect of a similar, but larger issue fixed in #773. The fix this, the maintainers of the redhat-postgresql-persistent chart will need to release a new version of their chart meeting the new naming expectations for Red Hat-contributed charts.

Their repo is here https://github.com/sclorg/helm-charts - @phracek can you look into bumping this chart's version and updating it to meet the Red Hat requirements (namely, the chart's name has to be redhat- prefixed), or otherwise point us to the right contact?

**Comment by phracek** [comment]
This issue should be fixed by #1660 . Feel free to close it. The upstream repository will be fixed later on.

**Comment by komish** [comment]
@strangiato let us know if you have any further issues
